src/Microsoft.PowerShell.Commands.Utility/commands/utility/AddType.cs

```diff
@@ -625,7 +625,7 @@ protected override void EndProcessing()
         // These dictionaries prevent reloading already loaded and unchanged code.
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-linux`
Add-Type.Throw if the type already exists
```powershell
Expected strings to be the same, but they were different.
Expected length: 64
Actual length:   60
Strings differ at index 0.
Expected: 'TYPE_ALREA...'
But was:  'COMPILER_E...'
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Add-Type.Tests.ps1: line 213
213:         $exc.FullyQualifiedErrorId | Should -BeExactly "TYPE_ALREADY_EXISTS,Microsoft.PowerShell.Commands.AddTypeCommand"
```